### PR TITLE
Update Winapp2.ini

### PR DESCRIPTION
**Pull request by APMichael**
- Removed dangerous entries and entries that are deleting user content. Such files should better be deleted manually.
- Removed non-functional messenger: Xfire (shut down 2016-04).
- Revised (unified) some entry names.

### CODE REVIEW

File: Winapp2.ini [hunk 1]
```diff
@@ -19211,16 +19093,6 @@ Default=False
...
-Section=Games
-Detect=HKLM\Software\Xfire
-Default=False
-Warning=Selecting this will delete any screen shots and videos you have taken, make sure you upload or move any you want to keep so they're not deleted.
-FileKey1=%CommonAppData%\Xfire|*.bak
-FileKey2=%CommonAppData%\xfire\videos|*.*
-FileKey3=%LocalAppData%\VirtualStore\ProgramData\Xfire|*.bak
-FileKey4=%LocalAppData%\VirtualStore\ProgramData\xfire\videos|*.*
-
 [Xfire Updates *]
```
Comment (MoscaDotTo) [inline Winapp2.ini]: We can probably remove any xfire entries as the servers shut down in 2016

**Comment by MoscaDotTo** [comment]
Some of these seem like contenders for winapp3.ini

File: Winapp2.ini (hunk 1 above)
Comment (APMichael) [reply Winapp2.ini]: Thanks for the information, another non-functional messenger! I will move both entries to the Archived entries.ini.

**Comment by APMichael** [comment]
Right, I will move them to the Winapp3.ini then.

**Pull request merged**